lib/ansible/galaxy/role.py

```diff
@@ -412,24 +421,42 @@ def install(self):
                                         # to debug a broken installation.
                                         if not n_part:
                                             continue
-                                        if n_part == '..':
+                                        if n_part == '..':  # TODO: only disallow .. for 'name' once symlinks are validated
+                                            invalid_parts |= True
```

Comment:
Isn't this the same as just
```suggestion
                                            invalid_parts = True
```
?

I keep having to re-verify what `|` does in the REPL whenever I see `|=`. It might be useful for collection types (sets, for example), but `|` used with `True` on any side of the operand always results in `True`. So why not just use the assignment operator to save the readers some potential confusion?

Reply:
fixed